scripts/autogen.py

```diff
@@ -1043,6 +1045,87 @@ def get_working_dir(arg):
     return arg[len("--working_dir=") :]
 
 
+
+def render_keras_nlp_tags(template):
```

Comment:
For the sake of clean factoring I recommend moving this to its own separate file